cvat-ui/src/components/annotation-page/top-bar/right-group.tsx

```diff
@@ -51,6 +56,37 @@ function RightGroup(props: Props): JSX.Element {
                 <Icon component={FullscreenIcon} />
                 Fullscreen
             </Button>
+            { jobInstance.guideId !== null && (
+                <Button
+                    type='link'
+                    className='cvat-annotation-header-guide-button cvat-annotation-header-button'
+                    onClick={async (): Promise<void> => {
+                        const PADDING = Math.min(window.screen.availHeight, window.screen.availWidth) * 0.4;
+                        const guide = await jobInstance.guide();
+                        Modal.info({
+                            icon: null,
+                            width: window.screen.availWidth - PADDING,
+                            content: (
+                                <>
+                                    <MDEditor
+                                        visibleDragbar={false}
+                                        data-color-mode='light'
+                                        height={window.screen.availHeight - PADDING}
+                                        preview='preview'
+                                        hideToolbar
+                                        value={guide.markdown}
+                                    />
+                                </>
+                            ),
+                        });
+
+                        // todo: handle loading, handle errors
```

Comment:
WIll general error handling be added in this version?

Reply:
Done